pages/linux/fdisk.md

```diff
@@ -0,0 +1,19 @@
+# fdisk
+
+> Manipulate disk partition table.
+
+- List  the  partition  tables for the sda disk and then exit:
+
+`fdisk -l /dev/sda`
+
+- Print the size (in blocks) of /dev/sda1:
+
+`fdisk -s /dev/sda1`
+
+- Create/remove/change partitions in the device:
+
+`fdisk /dev/sda`
+
+- Print help and then exit:
```

Comment:
Not sure that this is needed.

@agnivade / @waldyrious thoughts?

Reply:
Its fine. Some commands do have their -h/--help variants. So a user can see extra options with this.